rest-api/src/main/java/com/example/rest_api/RestApiApplication.java

```diff
@@ -0,0 +1,12 @@
+package com.example.rest_api;
```

Comment:
Please align package naming to https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names
which is the most commonly used approach

Reply:
updated it, thanks for letting me know.